django/apps/blue_management/blue_mgnt/views/utils.py

```diff
@@ -0,0 +1,24 @@
+
+OFFENDING_CSV_CHARS = set('@+-=|%')
+
+
+def is_injection(s):
+    return len(s) > 0 and s[0] in OFFENDING_CSV_CHARS
+
+
+def escape(payload):
+    '''
+    Escapes CSV payloads to prevent Excel injection.
+    Works only on str, and bypasses any other type.
+    '''
+    if isinstance(payload, str) and is_injection(payload):
+        payload = payload.replace("|", "\| ")
+        payload = "'" + payload + "'"
```

Comment:
Shouldn't the space be added here? Because the fact that you replace | only doesn't cover the case where it starts with =. Maybe I'm reading this wrong though.

Reply:
Yes, it makes sense here too, if for some reason quoting is removed, the victim would have still to strip the initial space: b6d2db1